<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMainDialog</class>
 <widget class="QDialog" name="frmMainDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>462</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>976</width>
    <height>462</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>976</width>
    <height>462</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>976</width>
    <height>462</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Jura</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>encFSGui 2.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>bitmaps/encfsgui.icns</normaloff>bitmaps/encfsgui.icns</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QFrame" name="verticalFrame">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>981</width>
     <height>87</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item>
     <widget class="QToolButton" name="btn_createVolume">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Shadow">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>231</red>
            <green>231</green>
            <blue>231</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Shadow">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>231</red>
            <green>231</green>
            <blue>231</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Shadow">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>231</red>
            <green>231</green>
            <blue>231</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Jura Medium</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>bitmaps/createfolder.png</normaloff>bitmaps/createfolder.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="popupMode">
       <enum>QToolButton::DelayedPopup</enum>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="btn_openVolume">
      <property name="font">
       <font>
        <family>Jura Medium</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add Existing</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>bitmaps/existingfolder.png</normaloff>bitmaps/existingfolder.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QToolButton" name="btn_browseVolume">
      <property name="font">
       <font>
        <family>Jura Medium</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>bitmaps/browsefolder.png</normaloff>bitmaps/browsefolder.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="btn_removeVolume">
      <property name="font">
       <font>
        <family>Jura Medium</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>bitmaps/removefolder.png</normaloff>bitmaps/removefolder.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="btn_infoVolume">
      <property name="font">
       <font>
        <family>Jura Medium</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Info</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>bitmaps/folderinfo.png</normaloff>bitmaps/folderinfo.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="btn_editVolume">
      <property name="font">
       <font>
        <family>Jura Medium</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>bitmaps/editfolder.png</normaloff>bitmaps/editfolder.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QToolButton" name="btn_mountVolume">
      <property name="font">
       <font>
        <family>Jura Medium</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Mount &amp;&amp; expose</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>bitmaps/mountfolder.png</normaloff>bitmaps/mountfolder.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="btn_unmountVolume">
      <property name="font">
       <font>
        <family>Jura Medium</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Unmount &amp;&amp; lock</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>bitmaps/unmountfolder.png</normaloff>bitmaps/unmountfolder.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="btn_unmountAll">
      <property name="font">
       <font>
        <family>Jura Medium</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Force unmount all</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>bitmaps/unmountfolder_all.png</normaloff>bitmaps/unmountfolder_all.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>15</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QToolButton" name="btn_Settings">
      <property name="font">
       <font>
        <family>Jura Medium</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>bitmaps/settings.png</normaloff>bitmaps/settings.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>15</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QToolButton" name="btn_Quit">
      <property name="font">
       <font>
        <family>Jura Medium</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>bitmaps/quit.png</normaloff>bitmaps/quit.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>981</width>
     <height>341</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTableWidget" name="tbl_Volumes">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>971</width>
      <height>331</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
  </widget>
  <widget class="QLabel" name="lbl_InfoLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>931</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jura</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>lbl_InfoLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
